en/firefox/browsers/compare/brave.ftl

```diff
@@ -0,0 +1,64 @@
+compare-brave-customization-of-ui-elements = Customization of UI elements and themes have been a favorite feature of { -brand-name-firefox } users for years and our avid community of developers have created a <a { $addons }>vast library</a> of open source add-ons and extensions allowing for even more personalization and functionality. Features that come with { -brand-name-firefox } when you download include our powerful <a { $screenshot }>screenshots tool</a>, accessibility features and integration with <a { $pocket }>{ -brand-name-pocket }</a> — a resource that lets users quickly save an article for later reading on any device.
+compare-brave-brave-also-supports-the-huge = { -brand-name-brave } also supports the huge library of extensions available in the { -brand-name-google } { -brand-name-chrome } web store and offers a variety of in-browser features like the aforementioned { -brand-name-brave } Rewards program, and support for downloading torrents in the browser.
+compare-brave-the-ability-to-sync-your-passwords = The ability to sync your passwords, extensions, form data, add-ons and other preferences across all your devices and operating systems is a feature that’s been available for years with { -brand-name-firefox }. The synced data is also encrypted, which means no one can access it from the outside.
+
+# Variables:
+#   $accounts (string) - link to /firefox/accounts/ with additional attributes for analytics
+#   $monitor (string) - link to monitor.firefox.com with additional attributes for analytics
+#   $breaches (string) - link to monitor.firefox.com/breaches with additional attributes for analytics
+compare-brave-the-firefox-browser-also-gives = The { -brand-name-firefox } browser also gives users the ability to sign up for a free <a { $accounts }>{ -brand-name-firefox-account }</a>. Having a { -brand-name-firefox } account is the key to unlocking syncing across devices, plus you get the added benefit of products like <a { $monitor }>{ -brand-name-firefox-monitor }</a> which monitors your email addresses and alerts you if any of your information has been involved in any known <a { $breaches }>data breaches</a>.
+
+# "Basic Attention Token" is a brand name.
+compare-brave-brave-also-recently-gained = { -brand-name-brave } also recently gained the ability to sync data across most popular operating systems and devices as well with the added capability of syncing your Basic Attention Tokens.
+compare-brave-when-comparing-the-two-browsers = When comparing the two browsers, both { -brand-name-firefox } and { -brand-name-brave } offer a sophisticated level of privacy and security by default, available automatically from the very first time you open them.
+compare-brave-overall-brave-is-a-fast-and = Overall, { -brand-name-brave } is a fast, and secure browser, that will have particular appeal to cryptocurrency users but for the vast majority of internet citizens, { -brand-name-firefox } remains a better and simpler solution.
```

Comment:
This sentence is long and disjointed, it seems. With the change of punctuation (removing a comma and adding a period, it reads better. May I suggest:
`... and secure browser that will have particular appeal to cryptocurrency users. But for the vast majority of...`